Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'solidus', '~> 1.4.0'
+gem 'solidus', github: 'solidusio/solidus', branch: 'master'
```

Comment:
we should change the solidus version based on an env variable, see [here](https://github.com/solidusio/solidus_auth_devise/blob/master/Gemfile) for an example.